spec/models/category_spec.rb

```diff
@@ -0,0 +1,33 @@
+require "rails_helper"
+
+RSpec.describe Category, type: :model do
+  let(:category) { Category.new(
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.